README.md

```diff
@@ -179,6 +179,11 @@ Here are all the types of checks currently implemented:
 - `big_brother_service` (`Metric::BigBrotherService`): Used in conjunction with [Big Brother](https://github.com/braintree/big_brother), reports health based on the overall health of another load balanced service.
   * service
 
+- `haproxy_backends_health` (`Metric::HaproxyBackendsHealth`): Checks UP/DOWN status across a set of servers in a Haproxy cluster and computes a weight based on how many are UP.
```

Comment:
"and computes a weight based on how many are UP."

perhaps something like this insted:

"and computes a litmus weight based on the HAProxy weights of the servers which have an UP status"?

Reply:
👍 4b086b85d1e25a2d80c5e057d75e2440e8c5907d